### PR TITLE
Fixes slime people (i don't know what the ill effect but runtimes is)

### PR DESCRIPTION
**Pull request by SandPoot**
## About The Pull Request
Proc calls without arguments, go figure.

## Why It's Good For The Game
More fixes!!!!!!!!!

## Changelog
:cl:
fix: Put arguments into handle_blood (it needs them).
/:cl:

### CODE REVIEW

**Comment by SandPoot** [comment]
Okay, hang on.

File: code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -9,7 +9,7 @@
 		. = FALSE
 	if(!.)
 		return
-	handle_blood()
+	handle_blood(seconds, times_fired)
```
Comment (Putnam3145) [inline code/modules/mob/living/carbon/life.dm]: i know it's out of scope but rename this var to delta_time i beg you (it's not actually necessary though)

**Review by Putnam3145 — APPROVED** [review]

File: code/modules/mob/living/carbon/life.dm (hunk 1 above)
Comment (SandPoot) [reply code/modules/mob/living/carbon/life.dm]: I would need to touch it elsewhere, as i'm just grabbing a proc argument from the mob subsystem

Comment (SandPoot) [reply code/modules/mob/living/carbon/life.dm]: Actually, it should probably be fine yeah, lemme see...

**Comment by SandPoot** [comment]
Rust-g download failed for integration tests.

**Pull request merged**